cellxgene_schema_cli/cellxgene_schema/migrate.py

```diff
@@ -349,6 +365,9 @@ def migrate(input_file, output_file, collection_id, dataset_id):
 
     # AUTOMATED, DO NOT CHANGE -- IF GENCODE UPDATED, DEPRECATED FEATURE FILTERING ALGORITHM WILL GO HERE.
     if DEPRECATED_FEATURE_IDS:
-        dataset = utils.remove_deprecated_features(dataset, DEPRECATED_FEATURE_IDS)
+        dataset = utils.remove_deprecated_features(adata=dataset, deprecated=DEPRECATED_FEATURE_IDS)
+
+    # Manually remap v38 terms to v44
+    dataset = utils.remap_deprecated_features(adata=dataset, remapped_terms=GENCODE_MAPPER)
 
-    dataset.write(output_file, compression="gzip")
```

Comment:
i'm not entirely sure what this change is - i didn't intentionally alter this line at least lol

Reply:
Every once in a while github makes mistakes in line-grouping and detecting diffs. I've seen it render an enormous diff for a file when in actuality only one line had changed.